plugins/action_files_cleanup/src/cleanupFiles.ts

```diff
@@ -23,15 +24,15 @@ interface ResponseValue {
   }[]
 }
 
-async function cleanupFiles(input: any, DBConnect: any) {
+// async function cleanupFiles(input: any, DBConnect: any) {
+async function cleanupFiles({ parameters, applicationData, DBConnect }: ActionPluginInput) {
   const db = databaseMethods(DBConnect)
 
-  const { applicationData } = input
-  const applicationSerial = input?.applicationSerial || applicationData.applicationSerial
-  const applicationid = input?.applicationId || applicationData.applicationId
+  const applicationSerial = parameters?.applicationSerial || applicationData?.applicationSerial
+  const applicationid = parameters?.applicationId || applicationData?.applicationId
   const {
     environmentData: { appRootFolder, filesFolder },
-  } = applicationData
+  } = applicationData as { environmentData: { appRootFolder: string; filesFolder: string } }
```

Comment:
I'm curious, why was this used?

Reply:
So the Action can use the full URL of the file -- the database only stores relative paths, as the host could change.